### PR TITLE
Comment section

### PR DESCRIPTION
**Pull request by LianTattersall**

### CODE REVIEW

**Comment by nc-niamh-smith** [comment]
I think it might be nice to have all the articles on the homepage as soon as I get to your website. Think about the way that other news sites work, or even apps such as reddit - they specifically have something for you to interact with as soon as you land on the page!

Really nice navigation, although I would like more styling on the comments to delineate them from each other and make it clear at a glance that they are comments rather than just bullet points of text.

I would also suggest that the article body text should probably be bigger than the comments text. Another thing to think about is the image size - when resizing the screen, it gets bigger until it takes over the screen - that's too much!

To summarise:

-consider what the user sees when they first arrive on your website
-your navigation to the article is great
-style the comments to look like comments
-style your single article to mimic what a user would expect
